docs/en/cloud/manage/scaling.md

```diff
@@ -6,18 +6,34 @@ slug: /en/manage/scaling
 
 # Automatic Scaling
 
-ClickHouse Cloud provides auto-scaling of your services. The scaling of ClickHouse Cloud Production services can be adjusted by organization members with the **Admin** role on the service **Settings** page.  Development services are always set to auto-scale and pause when idle.
+ClickHouse Cloud provides autoscaling for your services. The scaling of ClickHouse Cloud Production services can be adjusted by organization members with the **Admin** role on the service **Settings** page.
 
-![Settings page](images/AutoScaling.png)
+:::note
+Development services are always set to autoscale and will automatically pause when idle. Autoscaling settings for Development services cannot be changed.
```

Comment:
I suggest we take this out and say "Scaling only applies to Production services. Development services do not scale. You can choose to **upgrade** your service from Development to Production in order to scale it."